test/jdk/ProblemList.txt

```diff
@@ -495,7 +495,7 @@ java/lang/invoke/LFCaching/LFMultiThreadCachingTest.java        8151492 generic-
 java/lang/invoke/LFCaching/LFGarbageCollectedTest.java          8078602 generic-all
 java/lang/invoke/lambda/LambdaFileEncodingSerialization.java    8249079 linux-x64
 java/lang/invoke/RicochetTest.java                              8251969 generic-all
-java/lang/ref/OOMEInReferenceHandler.java                       8066859 generic-all
+java/lang/CompressExpandTest.java                               8287851 generic-all
```

Comment:
There may be merge issue here. JDK-8287851 was fixed last week and the CompressExpandTest.java removed from the exclude list at the same time.

Reply:
Thanks. fixed.